internal/backends/sqlite/database.go

```diff
@@ -218,8 +230,21 @@ func relationStats(ctx context.Context, db *fsql.DB, list []*metadata.Collection
 		return nil, lazyerrors.Error(err)
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/3293
-	stats.countIndexes, stats.sizeIndexes = 0, 0
+	stats.countIndexes = indexes
+
+	q = fmt.Sprintf(`
+		SELECT
+		    SUM(pgsize) AS SizeIndexes
```

Comment:
```suggestion
		SELECT SUM(pgsize)
```

To use the same formatting as WHERE. And the column name is unused